astroquery/esa/jwst/core.py

```diff
@@ -571,9 +571,12 @@ def resolve_target_coordinates(self, target_name, target_resolver):
         if target_resolver == "ALL" or target_resolver == "SIMBAD":
             try:
                 result_table = Simbad.query_object(target_name)
-                return SkyCoord((f'{result_table["RA"][0]} '
-                                 f'{result_table["DEC"][0]}'),
-                                unit=(units.hourangle,
+                # new simbad behavior does not return None but an empty table
+                if len(result_table) == 0:
+                    result_table = None
+                return SkyCoord((f'{result_table["ra"][0]} '
+                                 f'{result_table["dec"][0]}'),
+                                unit=(units.deg,
```

Comment:
This is going to result in an error, no?  `result_table["ra"]` is going to be `None["ra"]`.

There should be a test for this case

Reply:
THANKS! I forgot the return statement in the `if` :facepalm:

Reply:
And the test was there but with an indentation issue. Should be all fixed now, thanks again.